<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>showWin_pressureSensor</class>
 <widget class="QWidget" name="showWin_pressureSensor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>940</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>显示_压力传感器</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>显示——压力传感器</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_start_finish_mea">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>870</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>开始测量</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>870</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>电池电量：</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="pBar_battery">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QCustomPlot" name="plot_pressure" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>731</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>压力：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>640</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>供电回路电流：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>594</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>供电回路电压：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>684</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>信号回路电压：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>724</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>信号回路电流：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ch1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>500</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>通道1(标准传感器)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ch2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>500</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>通道2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ch3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>500</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>通道3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ch4">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>500</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>通道4</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ch5">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>500</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>通道5</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gridGroupBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>540</y>
     <width>691</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineE_signal_voltage_2"/>
    </item>
    <item row="4" column="4">
     <widget class="QLineEdit" name="lineE_signal_current_5"/>
    </item>
    <item row="4" column="0">
     <widget class="QLineEdit" name="lineE_signal_current_1">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="lineE_supply_current_3"/>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="lineE_pres_val_3">
      <property name="text">
       <string>40</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineE_pres_val_2">
      <property name="text">
       <string>30</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLineEdit" name="lineE_pres_val_1">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>20</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="lineE_pres_val_4">
      <property name="text">
       <string>50</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="lineE_signal_current_3"/>
    </item>
    <item row="4" column="3">
     <widget class="QLineEdit" name="lineE_signal_current_4"/>
    </item>
    <item row="3" column="3">
     <widget class="QLineEdit" name="lineE_signal_voltage_4"/>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineE_supply_voltage_3"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineE_supply_voltage_2"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineE_supply_current_2"/>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="lineE_supply_voltage_1">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="lineE_supply_current_4"/>
    </item>
    <item row="2" column="4">
     <widget class="QLineEdit" name="lineE_supply_current_5"/>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="lineE_supply_voltage_4"/>
    </item>
    <item row="1" column="4">
     <widget class="QLineEdit" name="lineE_supply_voltage_5"/>
    </item>
    <item row="3" column="4">
     <widget class="QLineEdit" name="lineE_signal_voltage_5"/>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="lineE_signal_voltage_1">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLineEdit" name="lineE_pres_val_5">
      <property name="text">
       <string>60</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineE_signal_current_2"/>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="lineE_supply_current_1">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="lineE_signal_voltage_3"/>
    </item>
    <item row="5" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineE_error_percentage_2"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineE_error_value_2"/>
      </item>
     </layout>
    </item>
    <item row="5" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineE_error_percentage_3"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineE_error_value_3"/>
      </item>
     </layout>
    </item>
    <item row="5" column="3">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="lineE_error_percentage_4"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineE_error_value_4"/>
      </item>
     </layout>
    </item>
    <item row="5" column="4">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLineEdit" name="lineE_error_percentage_5"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineE_error_value_5"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>564</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>压力值：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>820</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>液压站压力：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineE_hydra_val">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>810</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>760</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>误差：</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="plot_error" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>731</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>误差：</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
